### PR TITLE
sarkars/Initial commit for quantized custom op

### PR DESCRIPTION
**Pull request by sayantan-nervana**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -133,7 +133,7 @@ option(NGRAPH_DISTRIBUTED_ENABLE "Add distributed mode to the CPU backend" FALSE
 ExternalProject_Add(
     ext_ngraph
     GIT_REPOSITORY https://github.com/NervanaSystems/ngraph
-    GIT_TAG b5beac87b3c340ec40f902158a2fbd5edc247a1e
+    GIT_TAG nishant_quantize_bias
```
Comment (sayantan-nervana) [inline CMakeLists.txt]: This has to merge in ngraph

File: test/python/common.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def with_ngraph(self, l, config=tf.ConfigProto()):
             'NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS', None)
 
         os.environ['NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS'] = '1'
-        ngraph.enable()
+        ngraph.disable()
```
Comment (sayantan-nervana) [inline test/python/common.py]: change this back

File: CMakeLists.txt [hunk 3]
```diff
@@ -140,7 +140,7 @@ option(NGRAPH_DISTRIBUTED_ENABLE "Add distributed mode to the CPU backend" FALSE
 ExternalProject_Add(
     ext_ngraph
     GIT_REPOSITORY https://github.com/NervanaSystems/ngraph
-    GIT_TAG 516fbe13f24878a66b01f7a37a8d1ff166d84890
+    GIT_TAG 1286fd444ea6e981a2c7195ecf522d6fee1ca772
```
Comment (sayantan-nervana) [inline CMakeLists.txt]: ngraph has to be moved forward to get access to ngraph changes that nishant made

File: test/test_nn_ops.cpp [hunk 4]
```diff
@@ -1035,7 +1035,33 @@ TEST(NNOps, QuantizedMaxPool) {
...
+
+  vector<DataType> output_datatypes = {quant_type, DT_FLOAT, DT_FLOAT};
+
+  std::vector<Output> sess_run_fetchoutputs = {R.output, R.min_output,
+                                               R.max_output};
+  OpExecuter opexecuter(root, "QuantizedMaxPool", static_input_indexes,
+                        output_datatypes, sess_run_fetchoutputs);
+
+  opexecuter.RunTest();
+}
```
Comment (sayantan-nervana) [inline test/test_nn_ops.cpp]: this is a quantized max pool test, not a test for the op being added in this PR

File: CMakeLists.txt (hunk 3 above)
Comment (shresthamalik) [reply CMakeLists.txt]: Can we test the ./run-all-models script with the new version? whether all the 21 models are passing

**Comment by sayantan-nervana** [comment]
TESTNOW

**Pull request merged**